src/index.html

```diff
@@ -11,8 +11,53 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
```

Comment:
You have linked a SCSS file to your HTML, which is not correct. Browsers can't interpret SCSS, they only understand CSS. You need to compile your SCSS to CSS and then link the CSS file here.